.github/workflows/merge-python-requirements-upgrade-prs.yml

```diff
@@ -0,0 +1,87 @@
+name: merge-upgrade-prs
```

Comment:
Where possible, please use the [GitHub Action best practices](https://openedx.atlassian.net/wiki/spaces/AC/pages/3548282937/GitHub+Actions+best+practices). (and/or comment on them where they don't seem right to you...)